content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -82,17 +82,17 @@ The .NET Tracer supports Linux distributions as best-effort, based on minimum li
 - Alpine x64: [musl][21] 1.2.2 (from Alpine 3.14)
 - Alpine arm64: [musl][21] 1.2.4 (from Alpine 3.18)
 
-| Operating System         | Version | Architectures | Support level         | Package version              |
-| -------------------------|---------|---------------|-----------------------|------------------------------|
-| Alpine Linux (x64)       | 3.14+   |  x64,         | [GA](#support-ga)     | latest (.NET 5+ only, v1.27.0+) |
-| Alpine Linux (arm64)     | 3.18+   |  Arm64        | [GA](#support-ga)     | latest (.NET 6+ only, v3.2.0+) |
-| CentOS Linux             | 7+      |  x64          | [Maintenance](#support-maintenance)   | latest (EOL in v4.0.0)  |
-| CentOS Stream Linux      | 8       |  x64          | [Maintenance](#support-maintenance)   | latest (EOL in v4.0.0)  |
-| Debian                   | 10+     |  x64, Arm64   | [GA](#support-ga)     | latest                       |
-| Fedora                   | 29+     |  x64          | [GA](#support-ga)     | latest                       |
-| openSUSE                 | 15+     |  x64          | [GA](#support-ga)     | latest                       |
-| Red Hat Enterprise Linux | 7+      |  x64          | [GA](#support-ga)     | latest                       |
-| Ubuntu                   | 18.04+  |  x64, Arm64   | [GA](#support-ga)     | latest                       |
+| Operating System         | Version | Architectures | Support level                       | {{< tooltip text="SSI" tooltip="Single Step Instrumentation">}} Support | Package version |
```

Comment:
@andrewlock can you also validate this list?

Reply:
AFAIK we _only_ support ubuntu and Amazon Linux 2023 for SSI, but that's more of an SSI testing question than a .NET tracer question 🤔